src/Core/Framework/Routing/Facade/RequestFacade.php

```diff
@@ -76,7 +76,7 @@ public function scheme(): string
      */
     public function method(): string
```

Comment:
```suggestion
    public function method(): ?string
```

Update all the return types. The phpdoc comments as welll: `@return ?string`

Reply:
Hi David, I changed all that in my last commit as well. Thank you for letting me know.